cpp/include/cudf/io/text/data_chunk_source_factories.hpp

```diff
@@ -197,7 +241,7 @@ class string_data_chunk_source : public data_chunk_source {
   string_data_chunk_source(std::string const& data) : _data(data) {}
```

Comment:
This should probably take a `host_span` instead of a `string`. That also helps eliminate the extra string in the benchmark.